core/table.ts

```diff
@@ -450,12 +450,6 @@ export class Table {
     newTags.forEach(t => {
       this.proto.tags.push(t);
     });
-    if (!!this.uniqueKeyAssertion) {
```

Comment:
you can remove `uniqueKeyAssertion` and `mergedRowConditionsAssertion` as object members now, they're no longer useful